src/index.html

```diff
@@ -8,13 +8,34 @@
     <script type="module" src="./main.js"></script>
     <title>Film Book</title>
 
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
przez to się projekt nie buduje - zduplikowany kod. Coś poszło nie tak przy pullu